### PR TITLE
Fix reload

### PR DESCRIPTION
**Pull request by T4rk1n**
Hot Reload was broken by #108 with the removal of the Authentication component that wrapped the Reloader.

- Keep the config store state on soft reload.
- AppProvider returns `Loading...` if no configs as before #108

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [UNRELEASED] - 2018-12-19
+### Fixed
+[#110](https://github.com/plotly/dash-renderer/pull/110)
+
+- Keep the config store state on soft reload.
+- AppProvider returns `Loading...` if no configs as before #108
+
```
Comment (alexcjohnson) [inline CHANGELOG.md]: Merge all unreleased changes into the same block, right? Since once we *do* make a new release, they will all be part of it.

Comment (T4rk1n) [reply CHANGELOG.md]: I wanted to have the timestamp with the changes. It does look weird, will change.

**Comment by alexcjohnson** [comment]
Looks great 💃 
At some point it might be nice to write a test for hot reload, though offhand that sounds a bit painful...

**Comment by T4rk1n** [comment]
Yes, hot-reload really needs a test, atleast to make sure it still work.

Thinking of something like changing an assets css, wait for reload and assert the style has changed.

**Pull request merged**